database/Seeders/RolesTableSeeder.php

```diff
@@ -86,6 +86,20 @@ public function run()
                 'guard_name' => 'web',
                 'description' => 'User having this role will be able to Create, Update and Delete the clients having him as a key account manager.',
             ],
+            10 => [
+                'id' => 11,
+                'name' => 'project-manager',
+                'label' => 'Project Manager',
+                'guard_name' => 'web',
+                'description' => 'User having this role will be able to Create, Update and Delete the projects having him as a project manager.',
```

Comment:
I think this would be more exact, no?
```suggestion
                'description' => 'User having this role will be able to perform all operations in the projects module.',
```